app/components/task_list_items/base_component.rb

```diff
@@ -7,5 +7,9 @@ class BaseComponent < ViewComponent::Base
     def link_active?
       true
     end
+
+    def status_tag_component
+      StatusTags::BaseComponent.new(status: status)
+    end
```

Comment:
Make this the default so that if a task list item has a `status` method that returns a status the basic status tag component will be rendered automatically.

Reply:
My review_component is sitting in that directory but not using the base. Not sure if it should be moved out or changed to fit in.